site/weavedns.md

```diff
@@ -67,7 +77,7 @@ however specify an address in CIDR format manually. In this case you
 are responsible for ensuring that the IP addresses specified are
 uniquely allocated and not in use by any other container.
 
-Finally, WeaveDNS containers can be stopped with `stop-dns`.
+Finally, WeaveDNS containers can be stopped with `stop` or `stop-dns`.
```

Comment:
use similar language here as elsewhere, i.e. "in conjunction with... or independently..."